importer/tests/test_views.py

```diff
@@ -30,7 +33,53 @@ def test_upload_get(self):
     def test_upload_get_bad_repo(self):
         """GET upload page."""
         resp = self.client.get("/repositories/999/import", follow=True)
+        self.assertTrue(resp.status_code == NOT_FOUND)
+
+    def test_upload_with_permissions(self):
+        """GET upload page with different user permissions"""
+        self.logout()
+        self.login(self.USERNAME_NO_REPO)
+        # user has no permissions at all
+        resp = self.client.get("/repositories/{0}/import".format(
```

Comment:
Since the same URL is used multiple times in the test, maybe make it `import_url` and re-use it for clarity.